task/show-sbom/0.1/show-sbom.yaml

```diff
@@ -25,11 +25,20 @@ spec:
     - name: IMAGE_URL
       value: $(params.IMAGE_URL)
     script: |
-       #!/busybox/sh
-       cosign download sbom $IMAGE_URL 2>err
-       RET=$?
-       if [ $RET -ne 0 ]; then
-         echo Failed to get SBOM >&2
-         cat err >&2
-       fi
-       exit $RET
+      #!/busybox/sh
+      status=-1
+      max_try=5
+      wait_sec=2
+      for run in $(seq 1 $max_try); do
+        status=0
+        cosign download sbom $IMAGE_URL 2>err
```

Comment:
Do we want to print only the last error? We could collect all the errors with `2>>err`

Reply:
Good point. Updated.